### ISSUE TITLE
[Bug]: Checkcle Agent Not Detecting Docker Containers

### ISSUE THREAD

**Issue by kylehandy123** [operacle/checkcle#199]
### Environment

Unraid 7.1.4

### Bug Description

Checkcle server does not display running docker containers. 

### Steps to Reproduce

1. Go to Unraid Community Apps
2. Install Checkcle Agent
3. Fill out variable details
OR
1. Install directly via terminal with provided install command from Server

### Expected Behavior

The app should detect Docker containers running and display them in the Checkcle server.

### Logs

```shell
2025/12/06 18:20:20 Warning: Could not open log file: open /var/log/monitoring-agent/monitoring-agent.log: permission denied, using stdout only
2025/12/06 18:20:20 Successfully loaded environment file: /etc/monitoring-agent/monitoring-agent.env
2025/12/06 18:20:20 Environment check:
2025/12/06 18:20:20   - AGENT_ID: REDACTED
2025/12/06 18:20:20   - POCKETBASE_ENABLED: true
2025/12/06 18:20:20   - POCKETBASE_URL: http://192.168.1.201:6951
2025/12/06 18:20:20   - SERVER_NAME: unraid-01
2025/12/06 18:20:20   - HOSTNAME (detected): 5df663fae684
2025/12/06 18:20:20   - IP_ADDRESS (detected): 172.17.0.72
2025/12/06 18:20:20 Configuration validation passed
=== CheckCle Monitoring Agent Docker Entrypoint ===
Script starting at: Sat Dec  6 18:20:20 EST 2025
Running as user: uid=0(root) gid=0(root) groups=0(root),1(bin),2(daemon),3(sys),4(adm),6(disk),10(wheel),11(floppy),20(dialout),26(tape),27(video)
Container ID: 5df663fae684
Checking host directory mounts...
⚠ Docker socket not mounted - Docker monitoring disabled
  To enable Docker monitoring, add: -v /var/run/docker.sock:/var/run/docker.sock:ro
✓ Host monitoring mode enabled
✓ Host directories accessible
✓ Detected physical host: 5df663fae684
Creating environment file...
Using AGENT_ID from environment: REDACTED
Using SERVER_NAME from environment: unraid-01
Using POCKETBASE_URL from environment: http://192.168.1.201:6951
Using SERVER_TOKEN from environment
Using HOSTNAME from environment: 5df663fae684
Starting CheckCle Monitoring Agent for HOST SYSTEM monitoring...
Health check port: 8081
Physical host monitoring: ENABLED
Starting monitoring agent as monitoring-agent user...
```

### Checklist

- [x] I have searched existing issues to avoid creating duplicates.
- [x] I have provided sufficient information to reproduce the issue.
